tasks/redhat.yml

```diff
@@ -19,6 +19,16 @@
     - interfaces_route_tables | length > 0
   notify: Restart NetworkManager
 
+- name: RedHat | Add NetworkManager Custom Configuration
```

Comment:
That task description could have more detail, now it's description sort of tells me I can define that custom config somewhere and that custom config will get configured.

Reply:
I've adjusted to try an make it more obvious that these overrides are defined in the role. Feel free to suggest alternative wording. I've also adjusted the case to match the rest of the tasks.